app/models/concept-quiz.ts

```diff
@@ -0,0 +1,16 @@
+import { attr, belongsTo } from '@ember-data/model';
+import Model from '@ember-data/model';
+
+import type ConceptModel from 'codecrafters-frontend/models/concept';
+import type { Option } from 'codecrafters-frontend/models/concept-question';
+
+export default class ConceptQuiz extends Model {
```

Comment:
@libmartinito this can be removed, not used anywhere - I think we later renamed it to `ConceptQuestion` but didn't remove this model